### PR TITLE
Floyd Warshall Algorithm In Ruby

### PR DESCRIPTION
**Pull request by HardevKhandhar**
Fixes #2205 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Floyd Warshall Algorithm in Ruby
- 

#### Languages Used:
- Ruby
- 

#### Files Added:
- Floyd_Warshall_Algorithm.rb
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by HardevKhandhar** [comment]
Dear Ma'am @asha15 
The changes and modifications have been done. 
The commits are squashed.
Thank you.

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        i += 1
+    end
+    counter += 1
+end
+
+puts
+
+floyd_warshall(nodes, edge_matrix)
+
+=begin
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.rb:65]: keep a space after equal

**Comment by HardevKhandhar** [comment]
Dear Ma'am @asha15 
The suggested change is done.
The commits are squashed.
Thank you.

**Comment by asha15** [comment]
Keep going :)

**Pull request merged**